### PR TITLE
Add new Ohai deprecation pages

### PR DESCRIPTION
**Pull request by tas50**
Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: chef_master/source/deprecations_ohai_cloud.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+Deprecation: Cloud plugin replaced by the Cloud_V2 plugin (OHAI-8)
+===================================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/deprecations_ohai_cloud_v2.rst>`__
+
+In Ohai/Chef releases 13 we replaced the existing Cloud plugin with the Cloud_v2 plugin. This was done by having the Cloud_v2 plugin populate both node['cloud'] and node['cloud_v2']. The Cloud_v2 plugin includes a different data format that resolves many of the longstanding bugs in the existing Cloud plugin.
+
+Remediation
+=============
+
+If you have a cookbook that relies on data from node['cloud'] you will need to update the code to instead use node['cloud_v2']. This data will differ so you will want to run Ohai to see what modifications to your cookbook will be necessary with the new format.
```
Comment (thommay) [inline chef_master/source/deprecations_ohai_cloud.rst]: I don't think this remediation is right (or at least the wording is tricky), and it directly conflicts with the remediation for OHAI-11.  Can we provide some examples of exactly what changed here, and how one might think about making changes?

File: chef_master/source/deprecations_ohai_filesystem.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+Deprecation: Filesystem plugin replaced by the Filesystem V2 plugin. (OHAI-9)
+=============================================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/deprecations_ohai_filesystem.rst>`__
+
+In Ohai/Chef 13 we replaced the existing Filesystem plugin with the Filesystem2 plugin. This was done by having the Filesystem2 plugin populate both node['fileystem2'] and node['filesystem']. The Filesystem2 plugin includes a different data format that resolves many of the longstanding bugs in the Filesystem plugin.
+
+Remediation
+=============
+
+If you have a cookbook that relies on data from node['filesystem'] you will need to update the code to instead use node['fileystem2']. This data will differ so you will want to run Ohai to see what modifications to your cookbook will be necessary with the new format.
```
Comment (thommay) [inline chef_master/source/deprecations_ohai_filesystem.rst]: Same comments as for OHAI-8

**Review by thommay — CHANGES_REQUESTED** [review]

File: chef_master/source/deprecations_ohai_cloud.rst (hunk 1 above)
Comment (tas50) [reply chef_master/source/deprecations_ohai_cloud.rst]: Wording is just bad here. I'll rewrite this part

**Comment by davidwrede** [comment]
@tas50 @thommay Is this ready to merge?

**Comment by tas50** [comment]
This should be good to go

**Comment by davidwrede** [comment]
tenor-6775694

**Pull request merged**